config/samples/simplekafkacluster_with_one_way_ssl.yaml

```diff
@@ -0,0 +1,261 @@
+apiVersion: kafka.banzaicloud.io/v1beta1
```

Comment:
Let's not increase the number of sample KafkaCluster CRs as increases the maintenance burden. Add the `ssl.client.auth` config to `banzaicloud_v1beta1_kafkacluster.yaml`, `simplekafkacluster_ssl.yaml`  and `kafkacluster_with_external_ssl.yaml` examples commented out and with a short description. Users can look up the `ssl.client.auth` config in Apache Kafka documentation for more details.